downstream/modules/platform/ref-database-inventory-variables.adoc

```diff
@@ -0,0 +1,67 @@
+[id="ref-database-inventory-variables"]
+
+= Database variables
+
+[cols="50%,50%,50%",options="header"]
+|====
+| *RPM variable name* | *Container variable name* | *Description*
+| `pg_ssl_mode` | | Choose one of the two available modes: `prefer` and `verify-full`. 
+
+Set to `verify-full` for client-side enforced SSL. 
+
+Default = `prefer`
+| `postgres_ssl_cert` | `postgresql_tls_cert` | Location of the PostgreSQL SSL/TLS certificate.
```

Comment:
I think SSL/TSL are two different certificates? If that is the case try to avoid using forward slashes when referring to both, in most cases "or" or "and" slot in easy enough. Below is from the IBM style guide:

"In most cases, do not use slashes in text because they can be ambiguous. For example, 1/2 can mean one half, one and two, one of two, and one or two. Rewrite the text using and, of, or or."

Reply:
Oh yeah, I never know with this one because it conflicts with this statement: 
<img width="777" alt="Screenshot 2024-09-20 at 10 14 04" src="https://github.com/user-attachments/assets/cc8539b5-59a7-43cd-a149-acd9c0cf4da7">

Reply:
Ah if that is in our documentation, I feel that's a fair exception so!